### ISSUE TITLE
Unload button error

### ISSUE THREAD

**Issue by alfem** [dgtaheno/artillery-Genius#4]
After flashing your marlin firmware and updating TFT, if I click "unload" a message pops up: "unknow command: "M702 T0"

**Comment by dgtaheno** [comment]
Yes, just experienced it a few days ago, I'll check it!

**Comment by vertice3d** [comment]
Could you find a solution to this issue?

**Comment by dgtaheno** [comment]
Sorry for the long time without response on this issue @alfem and @vertice3d , I have been a bit busy lately.
I have investigated about the issue and I´m quite sure that it is a problem with dependencies in Marlin FW as mentioned here:

https://github.com/bigtreetech/BIGTREETECH-TouchScreenFirmware#marlin-dependencies

Despite I found the root cause, I need to find some time(energy) to correct the code, test it and update it.

Please be a little bit more patient.

Best regards.

**Comment by dgtaheno** [comment]
Hello @alfem & @vertice3d ,

I felt a bit ashamed and corrected the code now :-).

I just tested and now unload and load functions seem to work fine.

Good think is that with new TFT firmware, if already installed, you don´t need to open the printer anymore to upload the firmware.

Just go to  Menu->Settings->Connection->ON

upload the firmware via Cura(plenty manuals online and quite intuitive).

And you should be able to have it running quite fast.

Please keep reporting bugs and I´ll try to be faster.

Best regards.

**Comment by vertice3d** [comment]
Thank you very much for your help, I'll try it

**Comment by fjherna** [comment]
Hello All,
Yesterday I tried to follow the procedure to update the Firmware of my Artillery Genius.
It had already installed the new TFT firmware, so I thought that I don´t need to open the printer anymore to upload the firmware.
I just went to Menu->Settings->Connection->ON

I tried to update Firmware flashing with "Prusaslicer". However I always got the same message: 
               Using Port                    : /dev/ttyUSB0
                Using Programmer              : wiring
                Overriding Baud Rate          : 115200
avrdude-slic3r: stk500v2_recv(): timeout
avrdude-slic3r: stk500v2_getsync(): timeout communicating with programmer
avrdude-slic3r: Could not open port: /dev/ttyUSB0

Then I saw your message about the "baud rate" and I changed it from 250000 to  115200 at Menu->Settings->Connection->S. Ports

After that I continuing the same error messages from "PrusaSlicer".

At the end I decided to remove the cable from the TFT to the mainboard.

Then I repeated the process with "PrusaSlicer" and I was able to successfully flash the new Firmware.
:)

Thank you for it !!!

I know that I should been doing something wrong. However I was not able to find out what was the problem.

Regards,

fjherna1
